### DISCUSSION TITLE
model_cap_exceeded problem fix

### DISCUSSION THREAD

**Discussion by JaylonXiwei** [acheong08/ChatGPT#1211] [General]
Is there anyway to solve this error?
{"detail":{"message":"You have sent too many requests to the model. Please try again later.","code":"model_cap_exceeded","clears_in":7954}}

I have a plus account, and how should I use it with ChatGPT-Proxy-V4 together? Is there a tutorial on this?

Also, how could I continue my previous conversation? Now I need to start a new conversation everytime I run the program. Thanks!

**Comment by acheong08** [comment]
GPT-4 has cap of 25 messages per 3 hours by OpenAI
